File: Conscious Python - OOP/step 04. Специальные методы и атрибуты/4.2/4.2.01.py
'''

Допишите код так, чтобы на экран выводились слова, указанные в Sample Output. Для этого создайте метод __getattribute__ .

Sample Input:

Sample Output:

Superman
Batman
Spiderman

'''

class User:
    def __init__(self, name):
        self.name = name    

    # создайте метод __getattribute__
    def __getattribute__(self, item):
        return object.__getattribute__(self, item) + 'man'

user1 = User("Super")
user2 = User("Bat")
user3 = User("Spider")

print(user1.name, user2.name, user3.name, sep='\n')

File: Diving into JavaScript: For Beginners/step04/4.02/4.2.01.js
/*

Перед вами программа, которая должна запрашивать у пользователя число. 
Затем программа должна вывести все четные числа от 2 до введенного числа включительно. 
Впишите вместо многоточия правильную инструкцию.

Sample Input 1:

8

Sample Output 1:

2
4
6
8

Sample Input 2:

13

Sample Output 2:

2
4
6
8
10
12

*/

let n = Number(prompt("Введите целое число:"));

for (let i = 2; i <= n; i += 2) {
  console.log(i);
}
